### PR TITLE
Fix some typos (found by codespell)

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

File: Goobi/newpages/ProjekteBearbeiten.jsp [hunk 1]
```diff
@@ -223,7 +223,7 @@
 																	monthYearRowClass="monthYearRowClass"
 																	imageLocation="/newpages/images/calendarImages"
 																	popupButtonImageUrl="/newpages/images/calendarImages/calendar.gif" />
-																<%--popupButtonImageUrl="images/calender.gif"--%>
+																<%--popupButtonImageUrl="images/calendar.gif"--%>
```
Comment (henning-gerhardt) [inline Goobi/newpages/ProjekteBearbeiten.jsp]: A filename must not be correct in English language as long this file exists. If you want to translate the file name, you must rename this file too.

Comment (stweil) [inline Goobi/newpages/ProjekteBearbeiten.jsp]: This file does not exist in the source tree, neither as `images/calender.gif` nor as `images/calendar.gif`. I could also simply remove that comment line.

Comment (henning-gerhardt) [inline Goobi/newpages/ProjekteBearbeiten.jsp]: Then remove this comment.

Comment (stweil) [inline Goobi/newpages/ProjekteBearbeiten.jsp]: Done.

**Pull request merged**